tests/ha/ha_cluster_crash_test.pm

```diff
@@ -45,35 +45,52 @@ sub run {
     # List of things to check
     my @checks = qw(kill-sbd kill-corosync kill-pacemakerd split-brain-iptables);
     my $preflight_start_time = time;
-    my $fencing_start_delay = calculate_sbd_start_delay();
+    #
+    my $start_delay_after_fencing = calculate_sbd_start_delay();
 
     # Loop on each check
     foreach my $check (@checks) {
+        # reset fencing status form previous loop
+        $node_was_fenced = 0;
+
         # Execute the command
         my $cmd = "crm cluster crash_test --${check} --force";
         record_info("${check}", "Executing ${cmd}");
-        my $cmd_fails = script_run "${cmd}";
-        #record_info('ERROR', "Failure while executing '$cmd'", result => 'fail') unless (defined $cmd_fails and $cmd_fails == 0);
+        my $cmd_fails = script_run("${cmd}", die_on_timeout => 0);
```

Comment:
```suggestion
        my $cmd_fails = script_run($cmd, die_on_timeout => 0);
```
This is not Bash ;)  Other instances of `"${cmd}"` could be simplified.

Reply:
You are right, butI am innocent in this one, it was part of the old code :D
I removed it from other places as well.